controllers/suite_test.go

```diff
@@ -72,13 +80,20 @@ var _ = BeforeSuite(func() {
 	//+kubebuilder:scaffold:scheme
 
 	// start webhook server using Manager
+	webhookInstallOptions := &testEnv.WebhookInstallOptions
 	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
-		Scheme: scheme.Scheme,
+		Scheme:  scheme.Scheme,
+		Host:    webhookInstallOptions.LocalServingHost,
+		Port:    webhookInstallOptions.LocalServingPort,
+		CertDir: webhookInstallOptions.LocalServingCertDir,
 	})
 	Expect(err).NotTo(HaveOccurred())
 
 	// start reconcilers
 
+	err = (&dwsv1alpha1.Workflow{}).SetupWebhookWithManager(k8sManager)
```

Comment:
Aren't all these tests in the controller redundant to what was done in the webhook test suite?

Reply:
Two are redundant.  The first one I thought I'd leave in both.
I started putting them in the webhook test because that seemed logical, but then discovered I can't do much there without a status block, which doesn't happen if the reconciler isn't running so that's why I moved over there with the rest of the tests.

The second redundant test I'll remove from the webhook side...seems more convincing over on the controller side.